src/utils/registration-helpers.ts

```diff
@@ -880,3 +885,11 @@ export const convertToRegistrationSearchItem = (registration: Registration) => {
   };
   return registrationSearchItem;
 };
+
+/**
+ * Checks if the customer allows file upload for the given category.
+ * If 'true' all user roles are allowed to upload open files.
+ * If 'false' only users with curator role can upload open files.
```

Comment:
Sånn egentlig så bryr ikke denne funksjonen seg om user roles. Det er neste linje etter denne funksjonen som bestemer det. Kunne latt beskrivelsen her være om man drar inn `|| hasCuratorRole(user)` her, men da er vi tilbake til diskusjonen på kontoret 😅 Gjør det du føler er best, og så lar vi det være 😄 

```suggestion
 * Checks if the customer allows file upload for the given category.
```

Reply:
Jeg tenkte mest på hva `allowFileUploadForTypes` vil bety nå, og betydningen av den vil når være knytta til roller da. Så selv om denne i utgangspunktet ikke har noe forhold til rollene har output en del føringer for det tenker jeg 🤔 
Før var `false` at man absolutt ikke kunne ha filer
Nå er `false` at bare kuratorer skal kunne legge til åpne filer

Reply:
Tenker bare at det er en sjans for at denne kommentaren ikke oppdateres om bruken av funksjonen skulle endres. Men ser også verdien av å gi en føring på hvordan funksjonen skal brukes. Kjør på 🚀